### PR TITLE
Finish layout and routes

### PR DESCRIPTION
**Pull request by thonglhuet**

### CODE REVIEW

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hungnh103) [inline app/assets/javascripts/users.coffee]: remove file by using git command: `git rm <file-name>`

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  text-transform: uppercase;
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+  &:hover {
+    color: white;
+    text-decoration: none;
+  }
+}
+
```
Comment (hungnh103) [inline app/assets/stylesheets/custom.scss]: remove line

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (hungnh103) [inline app/assets/stylesheets/users.scss]: remove file by using git command `git rm <file-name>`

File: app/controllers/static_pages_controller.rb [hunk 4]
```diff
@@ -6,4 +6,6 @@ def help
   end
```
Comment (hungnh103) [inline app/controllers/static_pages_controller.rb]: insert blank line below this line

File: app/controllers/static_pages_controller.rb [hunk 5]
```diff
@@ -6,4 +6,6 @@ def help
...
   def about
   end
```
Comment (hungnh103) [inline app/controllers/static_pages_controller.rb]: as above

File: app/helpers/users_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (hungnh103) [inline app/helpers/users_helper.rb]: remove file by using git command `git rm <file-name>`

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```
Comment (hungnh103) [inline app/views/layouts/_footer.html.erb]: I18n
refer more at: http://guides.rubyonrails.org/i18n.html

Comment (ghost) [inline app/views/layouts/_footer.html.erb]: C tìm hiểu i18n để dùng nhé

File: app/views/layouts/_footer.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
```
Comment (ghost) [inline app/views/layouts/_footer.html.erb]: as above

File: app/views/layouts/_footer.html.erb [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (hungnh103) [inline app/views/layouts/_footer.html.erb]: coding convention: remove whitespace
refer more: https://github.com/framgia/coding-standards/blob/master/vn/README.md#ruby-on-rails

File: app/views/layouts/_footer.html.erb [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+      <li><%= link_to "Contact", contact_path %></li>
```
Comment (ghost) [inline app/views/layouts/_footer.html.erb]: as above

File: app/views/layouts/_header.html.erb [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+    <%= link_to "sample app", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",    root_path %></li>
+        <li><%= link_to "Help",    help_path %></li>
+        <li><%= link_to "Log in", '#' %></li>
+      </ul>
+    </nav>
+  </div>
+</header>
```
Comment (hungnh103) [inline app/views/layouts/_header.html.erb]: missing end file

File: app/views/layouts/_header.html.erb [hunk 12]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", root_path, id: "logo" %>
```
Comment (ghost) [inline app/views/layouts/_header.html.erb]: as above

File: app/views/layouts/_shim.html.erb [hunk 13]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+<script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+</script>
+<![endif]-->
```
Comment (hungnh103) [inline app/views/layouts/_shim.html.erb]: missing end file

File: app/views/layouts/application.html.erb [hunk 14]
```diff
@@ -1,14 +1,18 @@
...
-    <%= stylesheet_link_tag    'application', media: 'all',
-                                              'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-
-  <body>
-    <%= yield %>
-  </body>
+<head>
+  <title><%= full_title(yield(:title)) %></title>
```
Comment (hungnh103) [inline app/views/layouts/application.html.erb]: remove `()`

File: app/views/layouts/_header.html.erb (hunk 11 above)
Comment (ghost) [inline app/views/layouts/_header.html.erb]: missing end file

File: app/views/layouts/application.html.erb [hunk 15]
```diff
@@ -1,14 +1,18 @@
...
+  <%= csrf_meta_tags %>
+  <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (hungnh103) [inline app/views/layouts/application.html.erb]: use double quote `" "` in Rails
remove whitespace

Comment (ghost) [inline app/views/layouts/application.html.erb]: double quotes

File: app/views/layouts/application.html.erb [hunk 16]
```diff
@@ -1,14 +1,18 @@
...
+                             'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  <%= render 'layouts/shim' %>
+</head>
+<body>
+<%= render 'layouts/header' %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: as above

File: app/views/layouts/application.html.erb [hunk 17]
```diff
@@ -1,14 +1,18 @@
...
+<div class="container">
+  <%= yield %>
+</div>
+<%= render 'layouts/footer' %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: as above

File: app/views/layouts/application.html.erb [hunk 18]
```diff
@@ -1,14 +1,18 @@
...
-  </head>
-
-  <body>
-    <%= yield %>
-  </body>
+<head>
+  <title><%= full_title(yield(:title)) %></title>
+  <%= csrf_meta_tags %>
+  <%= stylesheet_link_tag    'application', media: 'all',
+                             'data-turbolinks-track': 'reload' %>
```
Comment (hungnh103) [inline app/views/layouts/application.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 19]
```diff
@@ -1,14 +1,18 @@
...
+  <%= render 'layouts/shim' %>
+</head>
+<body>
+<%= render 'layouts/header' %>
+<div class="container">
+  <%= yield %>
+</div>
+<%= render 'layouts/footer' %>
+</body>
 </html>
```
Comment (hungnh103) [inline app/views/layouts/application.html.erb]: missing end file

File: app/views/static_pages/contact.html.erb [hunk 20]
```diff
@@ -1,4 +1,4 @@
-<% provide(:title, "Contact") %>
+<% provide(:title, 'Contact') %>
 <h1>Contact</h1>
 <p>
   Contact the Ruby on Rails Tutorial about the sample app at the
```
Comment (hungnh103) [inline app/views/static_pages/contact.html.erb]: I18n

Comment (ghost) [inline app/views/static_pages/contact.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 21]
```diff
@@ -1,6 +1,14 @@
-<h1>Sample App</h1>
-<p>
-  This is the home page for the
-  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-  sample application.
-</p>
\ No newline at end of file
+<div class="center jumbotron">
```
Comment (hungnh103) [inline app/views/static_pages/home.html.erb]: please fix convention in this file

File: app/views/static_pages/home.html.erb [hunk 22]
```diff
@@ -1,6 +1,14 @@
...
+  <h1>Welcome to the Sample App</h1>
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: i18n

File: app/views/static_pages/home.html.erb [hunk 23]
```diff
@@ -1,6 +1,14 @@
...
+    This is the home page for the
+    <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    sample application.
+  </h2>
+
+  <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+</div>
+
+<%= link_to image_tag("rails.png", alt: "Rails logo"),
+            'http://rubyonrails.org/' %>
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: missing end file, i18n

File: config/routes.rb [hunk 24]
```diff
@@ -1,10 +1,7 @@
...
-  get 'static_pages/home'
-
-  get 'static_pages/help'
-
-  get  'static_pages/about'
-
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'static_pages#home'
-end
+  get  '/help',    to: 'static_pages#help'
```
Comment (ghost) [inline config/routes.rb]: double quotes

File: .idea/codeStyleSettings.xml [hunk 25]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectCodeStyleSettingsManager">
+    <option name="PER_PROJECT_SETTINGS">
+      <value />
+    </option>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default (1)" />
+  </component>
+</project>
```
Comment (hungnh103) [inline .idea/codeStyleSettings.xml]: missing end file

File: app/controllers/application_controller.rb [hunk 26]
```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  rotect_from_forgery with: :exception
   def hello
```
Comment (hungnh103) [inline app/controllers/application_controller.rb]: remove this method if you don't use it

File: app/views/static_pages/about.html.erb [hunk 27]
```diff
@@ -1,11 +1,16 @@
...
+  </a>
+  <%= t ".description_13" %>
+  <a href="http://www.railstutorial.org/book"><%= t ".description_14" %></a><%= t ".description_15" %>
+  <a href="http://screencasts.railstutorial.org/">
+    <%= t ".description_16" %>
+  </a>
+  <%= t ".description_17" %>
+  <a href="http://rubyonrails.org/"><%= t ".description_18" %></a>
+  <%= t ".description_19" %>
 </p>
```
Comment (hungnh103) [inline app/views/static_pages/about.html.erb]: missing end file

File: app/views/applicationcontroller/bar.html.erb [hunk 28]
```diff
@@ -1,2 +1,2 @@
-<h1>Applicationcontroller#bar</h1>
-<p>Find me in app/views/applicationcontroller/bar.html.erb</p>
+<h1><% t(".h1") %></h1>
```
Comment (hungnh103) [inline app/views/applicationcontroller/bar.html.erb]: e đặt tên biến I18n mang tính gợi nhắc nhé, sửa tương tự cho các biến còn lại

File: app/views/layouts/application.html.erb [hunk 29]
```diff
@@ -1,14 +1,19 @@
...
-  <head>
-    <title><%= full_title(yield(:title)) %></title>
-    <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag    'application', media: 'all',
-                                              'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-
-  <body>
+<head>
```
Comment (hungnh103) [inline app/views/layouts/application.html.erb]: indent

File: app/views/users/new.html.erb [hunk 30]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, t(".signup")) %>
```
Comment (hungnh103) [inline app/views/users/new.html.erb]: remove `()`

**Pull request merged**